source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,67 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, consider an active ``mc mirror --overwrite`` synchronizing content from Source to Destination.
+
+   If an object on Source changes, ``my mirror --overwrite`` synchronizes and overwrites any matching file on Destination.
+
+   Without ``--overwrite``, if an object already exists on the Destination, the mirror process fails for the object that exists in both locations.
```

Comment:
Nit - can simplify

```suggestion
   Without ``--overwrite``, if an object already exists on the Destination, the mirror process fails to synchronize that object.
```

Curiosity - do we fail and continue, or just fail?

Reply:
Fail and continue.